packages/@sanity/form-builder/src/inputs/BlockEditor-slate/FormBuilderBlock.js

```diff
@@ -253,21 +255,77 @@ export default class FormBuilderBlock extends React.Component {
     const value = this.getValue()
     const memberType = this.getMemberTypeOf(value)
 
+    const fieldsQty = (get(memberType, 'fields') || []).length
```

Comment:
using lodash.get is overkill here. I'd prefer `const fieldsQty = (memberType && memberType.fields || []).length`

Reply:
👍